input/fsh/profiles/FRCoreOrganizationProfile.fsh

```diff
@@ -19,7 +19,7 @@ Description: """Profile of the Organization resource for France. This profile sp
 * extension contains
     FRCoreOrganizationShortNameExtension named shortName 0..1 and
     FRCoreOrganizationDescriptionExtension named description 0..1 and
-    http://hl7.org/fhir/StructureDefinition/organization-period named usePeriod 0..1
+    https://hl7.org/fhir/StructureDefinition/organization-period named usePeriod 0..1
```

Comment:
error Cannot create usePeriod extension; unable to locate extension definition for: https://hl7.org/fhir/StructureDefinition/organization-period.
  File: /home/runner/work/hl7.fhir.fr.core/hl7.fhir.fr.core/igSource/input/fsh/profiles/FRCoreOrganizationProfile.fsh
  Line: 18 - 22
error No element found at path extension[usePeriod] for CardRule in FRCoreOrganizationProfile, skipping rule
  File: /home/runner/work/hl7.fhir.fr.core/hl7.fhir.fr.core/igSource/input/fsh/profiles/FRCoreOrganizationProfile.fsh
  Line: 18 - 22